shepherd_bms/include/compute.h

```diff
@@ -1,6 +1,23 @@
 #ifndef COMPUTE_H
 #define COMPUTE_H
 
-#include "datastructs.h"
```

Comment:
We probably want to keep this include in there, it's going to contain the data structures that we utilize to store data